.github/workflows/main.yml

```diff
@@ -51,31 +51,32 @@ jobs:
         echo "CC=mpicc" >> $GITHUB_ENV
         echo "CPPFLAGS=-I/usr/include/hdf5/mpich" >> $GITHUB_ENV
         echo "LDFLAGS=-L/usr/lib/x86_64-linux-gnu/hdf5/mpich" >> $GITHUB_ENV
+        echo "LD_LIBRARY_PATH=-L/usr/lib/x86_64-linux-gnu/hdf5/mpich" >> $GITHUB_ENV
```

Comment:
Shouldn't be needed, but if so the "-L" isn't needed. And does "/usr/lib/x86_64-linux-gnu/hdf5/mpich" contain libraries or is it "/usr/lib/x86_64-linux-gnu/hdf5/mpich/lib"?